resources/views/layouts/default/flights/table.blade.php

```diff
@@ -82,7 +82,7 @@ class="btn btn-sm btn-outline-primary">
             @endif
 
             <!-- If this flight has a briefing, show the link to view it-->
-            @if ($flight->simbrief)
+            @if ($flight->simbrief && $flight->simbrief->user_id === $user->id)
```

Comment:
Do we really need the user model here ? Can't we just use `Auth::user()->id`  instead of having the complete model ?

Reply:
I'd rather set `$user` because it can then be changed in the controller if needed. In a couple of places I reload the user with eager-loaded properties which don't come in the `Auth::user()` call

Reply:
Got the point. Just asking to learn as you know :)

Reply:
Of course!